clients/browser/index.html

```diff
@@ -404,7 +404,7 @@
 
         _balanceChanged(balance) {
             if (!balance) balance = Nimiq.Balance.INITIAL;
-            this._wltBalance.innerText = Nimiq.Policy.satoshisToCoins(balance.value).toFixed(Math.log10(Nimiq.Policy.SATOSHIS_PER_COIN));
+            this._wltBalance.innerHTML = '<span class="icon-nimiq"></span> ' + Nimiq.Policy.satoshisToCoins(balance.value).toFixed(Math.log10(Nimiq.Policy.SATOSHIS_PER_COIN));
```

Comment:
Shouldn't it work to just add the `icon-nimiq` class to the html element once you want to display the balance?